frontend/src/pages/commitmentsViz.js

```diff
@@ -17,12 +18,27 @@ import BubbleChart from '../components/visualization/BubbleChart';
 //    - Overview: https://gist.github.com/nbremer/667e4df76848e72f250b
 //    - Demo: https://nbremer.github.io/occupationscanvas/
 //  - Animated packed circles: https://bl.ocks.org/feifang/664c0f16adfcb4dea31b923f74e897a0
+//  - Canvas force graph: https://bl.ocks.org/mbostock/3180395
 
+const canvas = true;
 export default () => {
+  let visuals;
+  if (canvas) {
+    visuals = (
+      <>
+        <BubbleChartCanvas />
```

Comment:
Looks like this:
![Screen Shot 2020-04-26 at 12 43 46 PM](https://user-images.githubusercontent.com/2548712/80313869-b5fb4980-87bb-11ea-81f8-f13da30502c0.png)